scripts/npmPublish.sh

```diff
@@ -138,6 +138,12 @@ else
   fi
 
   # Luigi Oidc Plugin
+  if ( prepublishCheck "plugins/auth/public/auth-oidc-pkce" ); then
```

Comment:
Please remove it from the general release flow.
We wanted to create an own release cycle like container, client-support-ui5 or cra-release.